src/scripts/CMSSync/index.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs'
+import api from '../api'
+import settings from './settings'
+import { SettingsObject } from './types'
+import { getFormattedData } from './parser'
+
+const CMSSync = (config: SettingsObject[]) => {
+  config.forEach((item: SettingsObject) => {
+    console.info(` ✅ - ${item.name}`)
+    api(item.link).then((res) => {
+      const [, secondKey] = Object.keys(res)
```

Comment:
What is this `secondKey` ?

Reply:
The endpoint send us {total: 10, ifos: []}, changing the name of the second item always.

Reply:
We could use the category name for pool, ifos and farm, but sincerely I don't know if that has a name, does it?

Reply:
ok, maybe `data` ? it's generic, but this is what it is.